resources/nl-core/nl-core-Vaccination-event.xml

```diff
@@ -52,5 +86,41 @@
         <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-HealthProfessional-PractitionerRole" />
       </type>
     </element>
+    <element id="Immunization.reasonCode">
+      <path value="Immunization.reasonCode" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Immunization.reasonCode:vaccinationIndication">
+      <path value="Immunization.reasonCode" />
+      <sliceName value="vaccinationIndication" />
+      <short value="Why immunization occurred (person-specific)" />
+      <definition value="Describes the person-specific characteristics why the immunization is administered to this patient (e.g. asplenia, immunosuppression or diabetes), net immunization advice" />
+      <binding>
+        <strength value="required" />
+        <valueSet value="http://decor.nictiz.nl/fhir/4.0/ValueSet/2.16.840.1.113883.2.4.3.11.60.131.11.4" />
+      </binding>
+    </element>
+    <element id="Immunization.reasonCode:vaccinationMotive">
+      <path value="Immunization.reasonCode" />
+      <sliceName value="vaccinationMotive" />
+      <definition value="Displays the trigger based on what the vaccine was administered" />
+      <binding>
+        <strength value="required" />
+        <valueSet value="http://decor.nictiz.nl/fhir/4.0/ValueSet/2.16.840.1.113883.2.4.3.11.60.131.11.2" />
+      </binding>
+    </element>
+    <element id="Immunization.protocolApplied.targetDisease">

```

Comment:
This should probably be a slice and you should add the required metadata.

Reply:
I added the slice. Is the metadata correct/sufficient?

Reply:
Yes, looks good. I believe you still need to add some guidance on populating `.doseNumber`, right?